testsuite/tests/typing-gadts/unexpected_existentials.ml

```diff
@@ -102,7 +102,7 @@ let () =
 Line 2, characters 6-11:
 2 |   let Any x = Any () and () = () in
           ^^^^^
-Error: Existential types are not allowed in "let ... and ..." bindings,
+Error: Existential types are not allowed in let ... and ... bindings,
```

Comment:
This is an example that I think becomes substantially less readable without visible code markers.